continuous_integration/install_with_blis.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+CURRENT_PATH=$(pwd)
+
+# Assume Ubuntu: install a recent version of clang and libomp
+# echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main" | sudo tee -a /etc/apt/sources.list.d/llvm.list
+# echo "deb-src http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main" | sudo tee -a /etc/apt/sources.list.d/llvm.list
+# wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+# sudo apt update
+# sudo apt install clang-8 libomp-8-dev
```

Comment:
Is this still required? If not we can remove this comment.

Reply:
still wip, I'm updating test_nested_prange_blas to also test with BLIS.

I disabled clang temporarily but I'd like to do 2 jobs: one with clang gcc and another one with gcc clang.

Reply:
Also, by default, BLIS is sequential and num_threads will be -1 in the infos.
It can be a bit confusing since -1 means all cores sometimes, e.g. sklearn. I think we should map -1 to +1. wdyt ?